automation_tools/satellite6/upgrade/client.py

```diff
@@ -141,12 +141,16 @@ def satellite6_client_setup():
             int(clients_count)/2,
             host=docker_vm
         )[docker_vm]
+        # Adding sleep to freed up yum lock taken by above client generate task
+        time.sleep(30)
```

Comment:
Let's see how it goes w/ sleep otherwise we need to revisit again for more suitable fix..